src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -258,6 +258,19 @@ export class EcoverseService {
     return user;
   }
 
+  async addAdmin(user: IUser): Promise<IUser> {
+    const ctverse = await this.getEcoverse();
+    const adminsGroup = await this.userGroupService.getGroupByName(
+      ctverse,
+      'admins'
```

Comment:
group name should be gotten from Enum for safety

Reply:
Agreed and resolved.